src/main/steps/applicant1/hub-page/template.njk

```diff
@@ -37,13 +37,13 @@
 
     <div class="govuk-grid-column-one-third hub-right-column">
       <hr class="govuk-section-break govuk-section-break--l govuk-section-break--additional">
-      <h2 class="govuk-heading-s">{{ subHeading3 }}</h2>
+      <h2 class="govuk-heading-s">{{ iWantTo }}</h2>
```

Comment:
is this part different to the other page? if its the same this could be a common html component where we include it in both files.